mis_builder/models/aep.py

```diff
@@ -364,7 +364,18 @@ def do_queries(
                 ):
                     # in initial mode, ignore accounts with 0 balance
                     continue
-                self._data[key][acc["account_id"][0]] = (debit * rate, credit * rate)
```

Comment:
If self._data becomes `defaultdict(lambda: defaultdict(lambda: SimpleArray((0.0, 0.0))))`, this can become `self._data[key][acc["account_id"][0]] += (debit * rate, credit * rate)`

Reply:
SimpleArray behaves like a tuple, but with support for member-wise arithmetic operations.

I have pushed a commit here to illustrate.

(Note I actually wanted to do another PR, but I accidentally pushed to your branch. I hope you don't mind).

Reply:
Very handy this SimpleArray, thank you !